web/public/locales/en/common.json

```diff
@@ -508,32 +508,32 @@
-    "helpbuttonDescription": "A _helpbutton_ refers to a post. Every post published by a user in _helpbuttons_ is termed a _helpbutton_. _helpbuttons_ can represent offers, needs, exchanges, transport proposals, businesses, and so forth. They can also be renamed to suit the specific needs of your app.",
-    "communityDescription": "The _helpbuttons_ community is the backbone of this project, providing it with sustainability. This tool is designed for adaptability to a multitude of purposes, thereby benefitting a diverse range of users while also encouraging support for the project. By federating with other _helpbuttons_ apps, you can share users and resources. This method of collaboration enables us to unite more people and ultimately improve the tool we provide.",
-    "ethicsDescription": "At _helpbuttons_, we are committed to maintaining the highest standards of ethical conduct. We believe in transparency, honesty, and respect for all individuals. Our software is designed to connect people and facilitate sharing, which is a responsibility we take seriously. We encourage our users to use our platform with the same values in mind, prioritizing respectful communication and mindful sharing. Any misuse of the software, including exploitation, harm to others, or manipulation of the platform's functionalities for personal gain, will not be tolerated. Users are required to adhere to these ethics policies at all times.",
-    "privacyDescription": "At _helpbuttons_, we respect your privacy and are committed to protecting your personal data. Our privacy policy outlines how we collect, store, use, and disclose your personal information. We comply with all relevant privacy laws and regulations, ensuring your data is used solely for the purpose of providing and improving our service. We are transparent about our data collection processes, and you have the right to access, update, or delete your personal data at any time. Remember, sharing personal information in _helpbuttons_ posts is at your discretion—be mindful and only share what you are comfortable with others knowing.\nPlease direct your questions to the following email: help@helpbuttons.org",
-    "securityDescription": "Your security is our top priority. At _helpbuttons_, we implement stringent security measures to protect the integrity of our software and the safety of our users. We continually update and review our systems to ensure they meet the highest standards of cybersecurity. However, we also believe that security is a shared responsibility. We encourage our users to practice safe online behaviors, like using strong passwords and protecting personal information.\nPlease direct your questions to the following email: help@helpbuttons.org",
-    "contactDescription": "Please direct your questions to the following email: help@helpbuttons.org",
-    "networksEthicsDescription": "",
+    "helpbuttonsHighlight": "Helpbuttons",
```

Comment:
are you sure about this change?

Reply:
yes, because it refers to the software not to the post